full spelling and grammar check/and correction (#70)
MillenniumMachines/millenniummachines.github.io: docs/millennium-os/manual/chapters/40_gcodes.md

G27 [Zn]
```

> Used to park the spindle in a safe location, by default it will move the spindle to the top of the Z-Axis, trigger an `M5.9` (stop the spindle and wait for spin-down), and then move the table to the centre of the X-Axis and at the front (towards the operator) of the Y-Axis.

Reviewer: ```suggestion
Used to park the spindle in a safe location, by default it will move the spindle to the top of the Z-Axis, trigger an `M5.9` (stop the spindle and wait for spin-down), and then move the table to the centre of the X-Axis and at the front of the Y-Axis (towards the operator).
```

G37
```

> When using multiple milling tools, we must compensate for length differences between the tools. G37 can be used to (re-)calculate the length of the current tool about a reference surface or the previous tool.

Reviewer 2: "In relation to" seems clearer here

Reviewer: Agreed, about is fine when talking about rotation but in this case it is a relative length, calculated with the difference in height between the reference surface and toolsetter activation point.

G37.1
```

> When there is no toolsetter available, it is necessary to re-zero the Z origin after changing tools - because the new tool will never be installed with the same length as the previous one.

Reviewer: ```suggestion
When there is no toolsetter available, it is necessary to re-zero the Z-Axis origin after changing tools - because the new tool will never be installed with the same length as the previous one.
```

Reset the stored details for a given WCS, or the current WCS if not specified.

> Different fields are used for different types of probing operations, and we want to reset these values before running a probing cycle, so if previous values existed, but the probing cycle failed we would not end up using the formally valid values.

Reviewer 2: "previous" seems clearer here

M5011 W2
```

> Look up stored rotation values from the given workplace number (or the current workplace number if not specified), and if a rotation value is given, it will prompt the operator to apply it as a rotation compensation value using the inbuilt `G68` command.

Reviewer 2: "it" undefined: can we specify?

> Look up stored rotation values from the given workplace number (or the current workplace number if not specified), and if a rotation value is given, it will prompt the operator to apply it as a rotation compensation value using the inbuilt `G68` command.

Reviewer: ```suggestion
Look up stored rotation values from the given workplace number (or the current workplace number if not specified), and if a rotation value is given, the operator will be prompted to apply it as a rotation compensation value using the inbuilt `G68` command.
```

M6515 [Xnnn] [Ynnn] [Znnn]
```

> Provide at least one of X, Y, and Z coordinates to check that they are within the axes limits of the machine, and trigger an abort if they are outside the work envelope. This is used by other macros to make sure we do not try to move outside of machine limits.

Reviewer: I think maybe 'Accepts' would work better here. 'Provide' suggests we're talking from the perspective of the operator but then 'and trigger' is not something the operator is doing. The perspective should be consistent for both, or specified where unclear.

```suggestion
Accepts at least one of X, Y, and Z coordinates to check that they are within the axes limits of the machine, and trigger an abort if they are outside the work envelope. This is used by other macros to make sure we do not try to move outside of machine limits.
```

G8000
```

> Triggered when installing MillenniumOS for the first time, and can be called later to reconfigure MillenniumOS. It runs through a modal-driven configuration wizard prompting the user for all settings required to run MillenniumOS properly.

Reviewer: Clarifying what 'it' is.

```suggestion
Triggered when installing MillenniumOS for the first time, and can be called later to reconfigure MillenniumOS. This macro runs through a modal-driven configuration wizard prompting the user for all settings required to run MillenniumOS properly.
```

G6500 [Wnn]
```

> The guided Bore probe prompts the user for approximate diameter, overtravel, approximate center position, and probe depth.

Reviewer 2: Style seems inconsistent - can we potentially capitalise the name like so:

> - **Guided Bore Probe** prompts the user 

bold would help pick out the command your looking for

G6501 [Wnn]
```

> The guided Boss probe prompts the user for approximate diameter, clearance, overtravel, approximate center position, and probe depth.

Reviewer 2: inconsistent capitalisation

G6502 [Wnn]
```

> The guided Rectangle Pocket probe prompts the user for an approximate width (X), length (Y), overtravel and clearance, an approximate center position, and probe depth. Executes `G6502.1` with the relevant parameters to run the actual probe.

Reviewer 2: inconsistent capitalisation

G6503 [Wnn]
```

> The guided Rectangle Block probe prompts the user for an approximate width (X), length (Y), overtravel and clearance, approximate center position, and probe depth. Executes `G6503.1` with the relevant parameters to run the actual probe.

Reviewer 2: Inconsistent capitalisation

G6508 [Wnn]
```

> The guided Outside Corner probe prompts the user for an approximate width (X) and length (Y) of the 2 surfaces that make up the corner, a clearance and overtravel distance, a probing depth, a starting location, and the corner that we want to probe (front left, back right, etc). Executes `G6508.1` with the relevant parameters to run the actual probe.

Reviewer 2: inconsistent capitalisation

G6510 [Wnn]
```

> The guided Single Surface probe prompts the user for a starting location, overtravel distance, which surface to probe, a maximum probing distance, and for X and Y surfaces, a probing depth below the starting location. It can probe the Z height of a workpiece, or a single surface on X or Y if the operator knows these are aligned with the machine axes. This macro only probes a single point so it cannot calculate surface angles.

Reviewer 2: inconsistent capitalisation

G6520 [Wnn]
```

> The guided Vice Corner probing macro combines OUTSIDE CORNER and SINGLE SURFACE (Z) macros to zero all 3 axes of a WCS in a single probing operation. This macro prompts the user for the required parameters for the OUTSIDE CORNER macro, as well as a starting location. It calls the macros in sequence, probing the Z surface first before moving outwards and probing each X and Y surface that forms the corner.

Reviewer 2: inconsistent capitalisation

Reviewer: ```suggestion
The Vice Corner probing macro combines Outside Corner and Single Surface (Z) macros to zero all 3 axes of a WCS in a single probing operation. This macro prompts the user for the required parameters for the Outside Corner macro, as well as a starting location. It calls the macros in sequence, probing the Z surface first before moving outwards and probing each X and Y surface that forms the corner.
```

G73 F500 R-5 Q1 Z-10 X10 Y10
```

> Run a peck drilling with a partial retract cycle. **WARNING**: - this may not clear enough chips. You are likely better off using a drill cycle that retracts fully.

Reviewer: This extra `a` reads wrong to me - the cycle name is 'Peck Drilling with Partial Retract' so 'Run a "peck drilling with partial retract" cycle' is how this is intended to be read.

M7
```

> Enables the GPIO output associated with unpressurized coolant, set by the operator during the configuration wizard. If air blast (`M7.1`) is not already enabled, then this will be enabled before activating the coolant output.

Reviewer: ```suggestion
Enables the general-purpose output associated with unpressurized coolant, set by the operator during the configuration wizard. If air blast (`M7.1`) is not already enabled, then this will be enabled before activating the coolant output.
```

M8
```

> For those mad enough to build flood coolant into a DIY CNC machine, `M8` enables pressurized flood coolant on the GPIO output set by the operator during the configuration wizard.

Reviewer: `pressurised` is correct, `z` is an Americani(z)sation :)
